content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
```

Comment:
Is it clear that all the requirements for CVRs are applied to MACVRs as well, or should I explicitly state that?

Reply:
Unless we want to change MACVRs to being a subtype of CVRs, which would require a category for having subtypes of subtypes, it's probably best to add a rule saying "All other CVR rules apply" or some such thing.

Reply:
That is a great structural question. What do you think Gryphon? This would need to be an "indented sub type" to make it clear it inherits from CVRs.

Reply:
I think it may be a good idea to define MACVR only via inheritance.

> MACVR is both a MA and CVR.

Using inheritance means we don't have to duplicate language.

Reply:
And to be clear: There should be a separation between type definitions and type distributions. I recommend all type definitions get setup first. Then after that, stipulate type distributions.

Reply:
Yeah, I have Type Distributions in a completely separate place.

Reply:
How do we do inheritance for MACVR?

MA and CVR have conflicting requirements:
(a) CVR has == 1 answer, MA has > 1
(b) CVR is on word/phrase occurring in more than one verse in a chapter. MA has a unique word/phrase in first five words.


How do I word the inheritance?

I can't say something like "When the requirements of CVR and MA conflict, CVRs supersede" because then a MACVR can only have == 1 answer.

Reply:
Basically, a CR is a subclass of INT. It overrides parts of INT's rules. MA is a subclass of INT, overriding different rules. MACR is a subclass of MA and CR. Diamond inheritance. Now, how to describe that to non-Platonics... I've not had enough coffee today to offer a constructive suggestion...

Reply:
okay, that makes sense but that description will make everyone's eyes and ears bleed. 

so an MACR is "the CR aspects that override INTs, and the MA aspects that override INTs, combined" ??

Reply:
Well, start with CR and write something like, "A CR is an INT except A, B, and C." And then you can later write, "A MACR is both an MA and a CR except D, E, and F." And in both cases, you can add clarifying examples.

Reply:
okay, okay, simple enough. I was thinking you were saying through inheritance I could say a MACR is a "MA + CR" and that would be all that was needed, but there are problems with that.

Reply:
Cool. Yeah. My thinking on the rule book project is very much framed by software architecture patterns. So when I say inheritance, I'm never meaning purely inheritance unless I say so explicitly. It's possible my frame in wrong for a rule book project, but it feels right, just that the compiler is a human brain.